Remove unnecessary italics (#746)
apollographql/docs: src/content/graphos/cloud-routing/configuration.mdx

</Note>

By default, the router enables only GraphOS Studio to initiate browser connections to it. If your supergraph serves data to other browser-based applications, you need to do one of the following in the `cors` section of your **Router configuration YAML**:

- Add the origins of those web applications to the router's list of allowed `origins`.
  - Use this option if there is a known, finite list of web applications that consume your cloud supergraph.
- Add a regular expression that matches the origins of those web applications to the router's list of allowed `origins`.
  - This option comes in handy if you want to match origins against a pattern, see the example below that matches subdomains of a specific namespace.

> - Enable the `allowanyorigin` option.

Author: ```suggestion
- Enable the `allow_any_origin` option.
```

  - Use this option if your supergraph is a public API with arbitrarily many web app consumers.
  - With this option enabled, the router sends the [wildcard (`*`)](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Origin#directives) value for the `Access-Control-Allow-Origin` header. This enables any website to initiate browser connections to it (but they can't provide cookies or other credentials).

- You must use the `origins` + `match_origins` option if clients need to [authenticate their requests with cookies](#passing-credentials).

> The following snippet includes an example of each option (use either `allowanyorigin`, or `origins + match_origins`):

Author: ```suggestion
The following snippet includes an example of each option (use either `allow_any_origin`, or `origins + match_origins`):
```

```yaml
cors:
  # Set to true to allow any origin
  # (Defaults to false)

>   allowanyorigin: true

Author: ```suggestion
  allow_any_origin: true
```

  # List of accepted origins

>   # (Ignored if allowanyorigin is true)

Author: ```suggestion
  # (Ignored if allow_any_origin is true)
```

<Note>

> **Your router must specify individual `origins` to support credentialed requests.** If your router enables `allowanyorigin`, your browser will refuse to send credentials.

Author: ```suggestion
**Your router must specify individual `origins` to support credentialed requests.** If your router enables `allow_any_origin`, your browser will refuse to send credentials.
```

#
cors:
  # Set to true to allow any origin

>   allowanyorigin: false

Author: ```suggestion
  allow_any_origin: false
```

  # List of accepted origins

>   # (Ignored if allowanyorigin is set to true)

Author: ```suggestion
  # (Ignored if allow_any_origin is set to true)
```
